label_sync/labels.yaml

```diff
@@ -130,6 +130,12 @@ default:
       target: both
       prowPlugin: label
       addedBy: anyone
+    - color: c7def8
+      description: Categorizes issue or PR as related to a TEP (or needs a TEP).
+      name: kind/tep
+      target: both
```

Comment:
I wonder, if we use `kind/tep` on PRs, does that mean that we want to have a new section in release notes, or do we want to include PRs labelled as `kind/tep` under the `feature` group?

Reply:
right, I don't think we should use `kind/tep` in PRs, maybe it should indeed target only issues. Maybe it should be `area/tep` ? :sweat_smile:

Reply:
shall we make it
```
target: issues
description: Categorizes issue related to a TEP (or needs a TEP).
```

Reply:
maybe :upside_down_face:

Reply:
GitHub issues with `kind/tep` - this issue need/has a `TEP` for the feature request.
GitHub PRs with `kind/tep` in `Community` Repo where we propose a `TEP` - help classify PRs

We have been using `kind/feature` for implementing the feature being proposed in `TEP` which can/should stay as is. We do not want to change that to `kind/tep` as it could be misleading whether `TEP` is being proposed vs feature is being implemented.

Reply:
Thanks @pritidesai - that makes sense